javascript/ql/test/library-tests/Functions/undefinedreturns.js

```diff
@@ -0,0 +1,30 @@
+//semmle-extractor-options: --experimental
+
+
+
+//////////////////
+//              //
+//  GOOD CASES  //
+//              //
+//////////////////
+
+const fn_closure = function () 1;
+const arrowfn_w_expr_body = () => 1;
+function* generator_fn() { }
+async function async_fn() { }
+function fn_w_final_throw() { throw 1; }
+function* fn_w_final_yield() { yield 1; }
+function fn_w_final_return_w_expr() { return 1; }
+
+
+
+/////////////////
+//             //
+//  BAD CASES  //
```

Comment:
I don't think "good"/"bad" labels apply in this case, since you're not testing a query, just a library predicate.